recipe/meta.yaml

```diff
@@ -62,16 +63,17 @@ test:
     - jupyter lab path
     - jupyter server extension list
     - jupyter serverextension list
-    # disabled as these two tests aren't noarch compatible
-    # - jupyter server extension list | rg "jupyterlab.*{{ version.replace(".", "\\.") }}.*OK"
-    # - jupyter serverextension list | rg "jupyterlab.*{{ version.replace(".", "\\.") }}.*OK"
-    - jupyter lab build
+    - jupyter server extension list 1>server_extensions 2>&1
+    - cat server_extensions | rg "jupyterlab.*{{ version.replace(".", "\\.") }}.*OK"  # [not win]
+    - jupyter serverextension list 1>serverextensions 2>&1
+    - cat serverextensions | rg "jupyterlab.*{{ version.replace(".", "\\.") }}.*OK"   # [not win]
+    - jupyter lab build --dev-build=False --minimize=False
     - jupyter lab clean
-  downstreams:
+  #downstreams:
     # Additional testing for downstreams packages: ipympl, plotly, and pyviz_comms
-    - ipympl
-    - plotly
-    - pyviz_comms
+    #- ipympl
+    #- plotly
+    #- pyviz_comms
 
 about:
   home: https://github.com/jupyterlab/jupyterlab
```

Comment:
There is a redirect in the `doc_url` to the following URL:
https://jupyterlab.readthedocs.io/en/stable/

Reply:
I think the current `doc_url` (https://jupyterlab.readthedocs.io) is fine.